### PR TITLE
fix backward compatibility issues with django 3.0

### PR DESCRIPTION
**Pull request by rossi1**

### CODE REVIEW

**Comment by mirzadelic** [comment]
@rossi1 but should update python-dateutil or set >= because of other dependencies.

**Review by pcamelo — APPROVED** [review]

**Comment by pcamelo** [comment]
@jjkester could you review this PR, thanks in advance.

File: src/auditlog/filters.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 from django.contrib.admin import SimpleListFilter
 
 
+
```
Comment (jjkester) [inline src/auditlog/filters.py:4]: Blank line doesn't seem to be a necessary change

File: src/auditlog/models.py [hunk 2]
```diff
@@ -3,15 +3,18 @@
 import json
 import ast
 
+from six import python_2_unicode_compatible
+from six import iteritems, integer_types
+
```
Comment (jjkester) [inline src/auditlog/models.py:8]: These imports could be a bit neater

**Review by jjkester — CHANGES_REQUESTED** [review]
Thanks for the concise MR! Could you have a look at the following:

- Unnecessary blank lines
- Import formatting/ordering

**Comment by jjkester** [comment]
I have decided to go ahead with #257 to drop all unsupported Python versions, saves the trouble of using six.

Nevertheless, thanks for the effort of creating a PR.

File: src/auditlog/models.py (hunk 2 above)
Comment (rossi1) [reply src/auditlog/models.py:8]: I will fix this once I get on my computer

**Comment by paulpavlishcoupa** [comment]
> I have decided to go ahead with #257 to drop all unsupported Python versions, saves the trouble of using six.
> 
> Nevertheless, thanks for the effort of creating a PR.

Sounds good, I'll close the PR now.

File: src/auditlog/models.py (hunk 2 above)
Comment (rossi1) [reply src/auditlog/models.py]: Unnecessary blank lines removed and imports have been formatted properly

**Comment by jjkester** [comment]
Closed, as Django 3.0 support has been added in master. Release 1.0a1 is available. Python 2 is no longer supported, as it has been deprecated for a long time and support finally ended this January.

**Pull request closed without merging**